README.md

```diff
@@ -97,29 +97,49 @@ tokio-stream = { version = "0.1" }
 near-lake-framework = "0.6.1"
 ```
 
-## Cost estimates
+## Cost estimates (Updated Mar 10, 2022 with more precise calculations)
 
-**TL;DR** approximately $18.15 per month (for AWS S3 access, paid directly to AWS) for the reading of fresh blocks
+**TL;DR** approximately $20 per month (for AWS S3 access, paid directly to AWS) for the reading of fresh blocks
+
+### Historical indexing
+
+| Blocks | GET | LIST | Subtotal GET | Subtotal LIST | Total $ |
+|---|---|---|---|---|---|
+| 1000 | 5000 | 4 | 0.00215 | 0.0000216 | $0.00 |
+| 86,400 | 432000 | 345.6 | 0.18576 | 0.00186624 | $0.19 |
+| 2,592,000 | 12960000 | 10368 | 5.5728 | 0.0559872 | $5.63 |
+| 77,021,059 | 385105295 | 308084.236 | 165.5952769 | 1.663654874 | $167.26 |
```

Comment:
In addition to the pure count of blocks, I would add some human-readable comments ("1 day worth of blocks", "1 month", "since inception of NEAR mainnet")